src/stems/stem.js

```diff
@@ -21,9 +24,11 @@ function Stem(blockElem, onClick, containerElem) {
   this.addStyles();
 
   // Handle click event
-  this.stemButton.addEventListener( 'click', function() {
-    onClick();
-  }, false);
+  this.stemButton.addEventListener( 'click', function(me) {
+    return function(e) {
+      onClick(e, me);
+    };
+  }(this), false);
```

Comment:
this call-the-function-with-arg-immediately idiom is really only useful when dealing with closures inside for-loops. in this case,  i think

var me = this
addEventListener('click', function(e) {
  ..
})

is clearer (no tested func).